submission_rules.adoc

```diff
@@ -227,19 +227,30 @@ The submission process defines how to submit code and results for review and eve
 
 Submitters must register with the submitters working group and begin attending meetings at least **eight weeks before the deadline. **In order to register, a submitter or their org must sign the relevant CLA and provide primary and secondary github handles and primary and secondary POC email address.
 
+### How to Submit
 
-### Github repo
+The goal of the submission process is to ensure a successful submission for as many submitters as possible in a fair manner. Therefore, the submission process is structured to ensure that submissions are well formed. 
 
-MLPerf will provide a private Github repository for submissions. Each submitter will submit one or more pull requests containing their submission to the appropriate Github repo before the submission deadline. Pull requests may be amended up until the deadline. 
+A submission is made by placing an encrypted tarball in a MLCommons-provided cloud storage bucket and confirming the submission using an MLCommons web UI.
 
+MLCommons provides a cloud storage bucket [TODO: URL] for submitting encrypted tarballs up to fourteen days before the deadline. Submitters are encouraged to submit as early as possible during this period, since their results will not be visible to others and they will have a chance to fix any issues.
 
-### Licensing
+MLCommons provides a web UI [TODO: URL] for verifying scores contained in the tarball. When provided { private key, file name }, the UI decrypts, untars, and runs a submission verifier then displays results or errors. The submitter may confirm results as final and receive an email receipt. *All submissions must be confirmed in this manner or they will be disregarded.*
+
+### Late Submissions
+
+#### Post-submission grace period (submission deadline + 60 minutes):
+MLPerf will allow submissions for up to 60 minutes after the published deadline without explanation or penalty. This grace period will be advertised as little as possible. The 60 minute limit will be strictly enforced.
```

Comment:
I'm a little uncomfortable with the line:
> This grace period will be advertised as little as possible.
New submitters insufficiently familiar with the rules or this update might give up on the dot, only to realise later they could have persevered and submitted within 60 minutes. I'd suggest to remove this sentence, and make sure that this is explicitly mentioned in the new submitters' orientation.